packages/charts/src/gauge/gauge-util.ts

```diff
@@ -202,29 +248,30 @@ export class GaugeUtil {
     /**
      * Generates rendering tools for standard gauge attributes
      *
+     * @param gaugeConfig The configuration for the gauge
      * @param mode The mode of the gauge (Donut, Horizontal, or Vertical)
      *
      * @returns {IGaugeRenderingTools} The rendering tools for standard gauge attributes
      */
-    public static generateRenderingTools(mode: GaugeMode): IGaugeRenderingTools {
+    public static generateRenderingTools(gaugeConfig: IGaugeConfig, mode: GaugeMode): IGaugeRenderingTools {
         const renderingTools: Record<GaugeMode, IGaugeRenderingTools> = {
             [GaugeMode.Donut]: {
                 mainRendererFunction: () => new RadialRenderer(donutGaugeRendererConfig()),
-                thresholdsRendererFunction: () => new DonutGaugeThresholdsRenderer(),
+                thresholdsRendererFunction: () => new DonutGaugeThresholdsRenderer({ enabled: !gaugeConfig.disableThresholdMarkers }),
```

Comment:
Could you explain what's happening here exactly why are we flipping the boolean?